cms/templates/js/no-groups.underscore

```diff
@@ -0,0 +1,3 @@
+<div class="no-group-configurations-content">
+    <p><%= gettext("You haven't created any cohort groups yet.") %><a href="#" class="button new-button"><i class="icon-plus"></i> <%= gettext("Add your first Cohort Group") %></a></p>
```

Comment:
Terminology:
"You have not created any cohorted content groups yet."
"Add your first Cohorted Content Group".